components/user/UserTabs.js

```diff
@@ -32,20 +33,36 @@ export default function UserTabs({ tabs, setTabs }) {
         <div className="border-b border-gray-200">
           <nav className="-mb-px flex" aria-label="Tabs">
             {tabs.map((tab) => (
+              <div className="inline-block relative w-1/4"  key={tab.name}>
               <a
-                key={tab.name}
                 href={tab.href}
                 onClick={(e) => changeTab(e, tab.name)}
                 className={classNames(
                   tab.current
                     ? "border-indigo-500 text-indigo-600"
                     : "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300",
-                  "w-1/4 py-4 px-1 text-center border-b-2 font-medium text-sm"
+                  "block py-4 px-1 text-center border-b-2 font-medium text-sm"
                 )}
                 aria-current={tab.current ? "page" : undefined}
               >
                 {tab.name} ({tab.total})
               </a>
+              {tab.current ? <BiSortAlt2
```

Comment:
I think `tab.current && <BiSortAlt2` would be easier to read and then the ` null` is now required